ide/utils/sdk/sdk_scripts.py

```diff
@@ -298,9 +298,9 @@ def build(ctx):
 
     ctx.set_group('bundle')
     ctx.pbl_bundle(binaries=binaries, js=ctx.path.ant_glob(['src/pkjs/**/*.js', 'src/pkjs/**/*.json']), js_entry_file='src/pkjs/{{pkjs_entry}}')
-"""
+""".replace('{{pkjs_entry}}', project.pkjs_entry_point)
```

Comment:
No, my concern was not with the jshint parameter, but rather the pkjs_entry one. However, given that pkjs_entry being None is no longer an issue, I think chaining the pkjs replace with the existing jshint replace is sufficient.